draft-tigress-requirements.md

```diff
@@ -149,7 +149,6 @@ General terms:
 - (Req-UnderstoodFormat) Both Sender application and Receiver application MUST be able to recognize the format.
 - (Req-RoundTrips) Solution SHALL allow for multiple round trips or multiple reads/writes between one set of Sender and Receiver devices.
 - (Req-Preview) Solution SHOULD allow for extensibility and discoverable extensions (preview of share invitation).
-- (Req-RedemptionHandling) Shared Provisioning Information SHOULD route Receiver to redeem Provisioning Information using the designated Credential Management Application (e.g. Wallet).
```

Comment:
The goal of this requirement is to say that the solution must support some kind of universal linking / deep linking. Which url based or file type based solutions could support. This is kind of application behavior, but we need the protocol to not preclude this behavior.

Reply:
I believe, in a nutshell, this is an addition to the requirement of preview extension - once a user receives a shareURL in a message, they should be able to see a preview and easily forward the link to the proper application by tapping the preview image - this would require a support on the OS level (ref to deep-linking)

Reply:
I agree, that "(Req-UnderstoodFormat)" requirement covers this.